docs/source/reference/integration.rst

```diff
@@ -13,6 +13,9 @@ Integration
     :members:
     :exclude-members: infer_relative_search_space, sample_relative, sample_independent
 
+.. autoclass:: IgnitePruningHandler
```

Comment:
How about adding `IgnitePruningHandler` to the [Integration Modules for Pruning](https://github.com/pfnet/optuna/blob/master/docs/source/tutorial/pruning.rst#integration-modules-for-pruning)  list in the pruning tutorial.

Reply:
Thank you for your suggestion. I added it as follows:
![image](https://user-images.githubusercontent.com/3255979/66177471-83152b00-e69c-11e9-80c1-4fc0d827c07e.png)

Corresponding commit: https://github.com/pfnet/optuna/pull/561/commits/bae4561bbdbe69f0197a89486fe0e105690c7b93

Reply:
Thanks.